RealmTasks Apple/RealmTasks iOS/CellPresenter.swift

```diff
@@ -32,35 +32,37 @@ class CellPresenter<Item: Object where Item: CellPresentable> {
     }
 
     func deleteItem(item: Item) {
-        viewController.uiWrite {
-            guard !(item as Object).invalidated, let index = items.indexOf(item) else {
+        try! items.realm?.write {
```

Comment:
the whole idea behind `uiWrite` is to consolidate common things we want to do when doing interface-driven writes, such as the call to `didUpdateList` you've added a little further. Why did this change from `viewController.uiWrite` to `try! items.realm?.write`?